app/routes/index.js

```diff
@@ -216,58 +242,58 @@ module.exports = function(app, fs) {
 		.get(auth, function(req, res) {
 			res.render(path + '/public/user');
 		});
```

Comment:
Let's do away with the GET and POST endpoints of '/user'. We don't need it anymore.

Reply:
Again, was just trying out stuff with image upload. Removed.